modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/portlet/SearchPortletSearchResultPreferences.java

```diff
@@ -41,8 +40,8 @@ public SearchPortletSearchResultPreferences(
 	}
 
 	@Override
-	public Optional<String> getFieldsToDisplayOptional() {
-		return Optional.empty();
+	public String getFieldsToDisplay() {
+		return StringPool.BLANK;
```

Comment:
@julschong Please move this change to last commit, this is not a usage you need to apply to.